shared/teams/team/index.js

```diff
@@ -74,57 +70,117 @@ const Help = isMobile
       </Box>
     )
 
-class Team extends React.PureComponent<Props> {
-  _renderItem = (index: number, item: RowProps) => {
-    return (
-      <Box
-        key={item.username}
-        style={{
-          ...globalStyles.flexBoxRow,
-          alignItems: 'center',
-          flexShrink: 0,
-          height: isMobile ? 56 : 48,
-          padding: globalMargins.tiny,
-          width: '100%',
-        }}
-      >
-        <Avatar username={item.username} size={isMobile ? 48 : 32} />
-        <Box style={{...globalStyles.flexBoxColumn, marginLeft: globalMargins.small}}>
-          <Text type={this.props.you === item.username ? 'BodySemiboldItalic' : 'BodySemibold'}>
-            {item.username}
-          </Text>
-          <Box style={globalStyles.flexBoxRow}>
-            {!!showCrown[item.type] &&
-              <Icon
-                type="iconfont-crown"
-                style={{
-                  color: globalColors.black_40,
-                  fontSize: isMobile ? 16 : 12,
-                  marginRight: globalMargins.xtiny,
-                }}
-              />}
-            <Text type="BodySmall">{typeToLabel[item.type]}</Text>
-          </Box>
-        </Box>
-      </Box>
+type TeamTabsProps = {
+  admin: boolean,
+  members: Array<MemberRowProps>,
+  requests: string[],
+  loading?: boolean,
+  selectedTab?: string,
+  setSelectedTab: (?Constants.TabKey) => void,
+}
+
+const TeamTabs = (props: TeamTabsProps) => {
+  const {admin, members, requests, loading = false, selectedTab, setSelectedTab} = props
+  let membersLabel = 'MEMBERS'
+  membersLabel += !loading || members.length !== 0 ? ' (' + members.length + ')' : ''
+  const tabs = [
+    <Text
+      key="members"
+      type="BodySmallSemibold"
+      style={{
+        color: globalColors.black_75,
+      }}
+    >
+      {membersLabel}
+    </Text>,
+  ]
+  if (admin) {
+    const requestsLabel = `REQUESTS (${requests.length})`
+    // TODO Pending invite tab
+    tabs.push(
```

Comment:
Can't this just be `tabs.push(<Text ...)`?

Reply:
Yep, fixed